anitya/db/migrations/versions/b13662e5d288_add_admin_flag.py

```diff
@@ -0,0 +1,30 @@
+"""Add admin flag
+
+Revision ID: b13662e5d288
+Revises: ac10bf3f974c
+Create Date: 2018-11-14 10:02:06.593605
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b13662e5d288'
+down_revision = 'ac10bf3f974c'
+
+
+def upgrade():
+    """ Add 'admin' flag to users table. """
+    op.add_column(
+        'users',
+        sa.Column(
+            'admin',
+            sa.Boolean,
+            default=False)
+    )
```

Comment:
This will remove the admin permission to the users that are currently admin. You might want to make sure that the current admins have the value set to `True` during the migration.

Reply:
Current admins are defined in the configuration file, so this shouldn't change anything.